tests/gui/test_DirectOptionMenu.py

```diff
@@ -0,0 +1,70 @@
+from direct.gui.DirectOptionMenu import DirectOptionMenu
+
+def test_menu_destroy():
+    menu = DirectOptionMenu(items=["item1", "item2"])
+    menu.destroy()
+
+def test_showPopupMenu():
+    menu = DirectOptionMenu()
+    exceptionThrown = False
+    try:
+        menu.showPopupMenu()
+    except:
+        # Showing an option menu without items will raise an exception
+        exceptionThrown = True
+    assert exceptionThrown
```

Comment:
I'll fix this for now, but for future reference, pytest has a more idiomatic way to do this:
```python
with pytest.raises(Exception):
    menu.showPopupMenu()
```